saleor/static/dashboard-next/icons/Moon.tsx

```diff
@@ -4,18 +4,18 @@ import * as React from "react";
 export const Moon = createSvgIcon(
   <>
     <path
-      fill-rule="evenodd"
-      clip-rule="evenodd"
+      fillRule="evenodd"
```

Comment:
I know it's not an issue of this PR but haven't you consider using svg-sprites ( and in final getting something similar to this `<CustomIcon name='Moon' />`) instead of adding new icon as a separate component each time?

Reply:
Yeah, I'm gonna do it in some maintenance sprint.